package.json

```diff
@@ -9,7 +9,7 @@
     "clean": "dune clean",
     "test": "node ./scripts/run_integration_tests.js",
     "install:examples": "(cd examples/typescript-react-example && npm install) & (cd examples/flow-react-example && npm install) & (cd examples/untyped-react-example && npm install) & (cd examples/commonjs-react-example && npm install)",
-    "build:examples": "(cd examples/typescript-react-example && npm run clean && npm run build) && (cd examples/flow-react-example && npm run clean && npm run build) & (cd examples/untyped-react-example && npm run clean && npm run build) & (cd examples/commonjs-react-example && npm run clean && npm run build)",
+    "build:examples": "(cd examples/typescript-react-example && npm run clean && npm run build) && (cd examples/flow-react-example && npm run clean && npm run build) && (cd examples/untyped-react-example && npm run clean && npm run build) && (cd examples/commonjs-react-example && npm run clean && npm run build)",
```

Comment:
This is intentional, to analyze the examples in parallel.

Reply:
Oh ok I'll revert – I changed it because it was causing my terminal to hang, and the inconsistency of single/double ampersands made it look like a mistake!